ambiata-p.cabal

```diff
@@ -18,6 +18,7 @@ library
                         , bifunctors                  >= 4.2        && <= 5.3
                         , deepseq                     >= 1.3        && <= 1.5
                         , semigroups                  >= 0.16       && < 0.19
+                        , text                        >= 1.1
```

Comment:
I suspect we've tried to avoid this in the past, but it would certainly be very convenient.